code/modules/clothing/head/misc.dm

```diff
@@ -175,7 +175,25 @@
 	name = "fedora"
 	icon_state = "fedora"
 	item_state = "fedora"
-	desc = "A really cool hat if you're a mobster. A really lame hat if you're not."
+	desc = "It's NOT a trilby!"
+	action_button_name = "Tip Fedora"

```

Comment:
No, this is deprecated, give it a proper ```/action``` datum and put its type in this object's ```actions_types ``` list